src/NServiceBus.Core.Tests/Serializers/Json/JsonMessageSerializerTest.cs

```diff
@@ -9,8 +9,33 @@ namespace NServiceBus.Serializers.Json.Tests
     using MessageInterfaces.MessageMapper.Reflection;
     using NUnit.Framework;
 
+    class MessageWithException
+    {
+        public Exception Blah { get; set; }
```

Comment:
@timbussmann I've tried to make the test more focused. Also added a test for the case where an interface message contains an invalid prop

https://github.com/Particular/NServiceBus/pull/4133/commits/c01218679c7b9613eb6231f282f367ba4092bf54

thoughts?